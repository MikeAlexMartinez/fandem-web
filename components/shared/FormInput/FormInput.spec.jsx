import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import FormInputComponent from './FormInput';

describe('<FormInput />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(
      <FormInputComponent value="test value">
        {(formState, { onChange, onBlur }) => (
          <input {...formState} onChange={onChange} onBlur={onBlur} />
        )}
      </FormInputComponent>,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
