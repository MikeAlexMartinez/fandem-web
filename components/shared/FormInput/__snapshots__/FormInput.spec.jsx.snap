// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormInput /> renders and matches the snapshot 1`] = `
<input
  errors={null}
  hasChanged={false}
  isValid={true}
  onBlur={[Function]}
  onChange={[Function]}
  originalValue="test value"
  touched={false}
  value="test value"
/>
`;
